plugins/xenserver/xenapi/contrib/rpmbuild/SPECS/openstack-xen-plugins.spec

```diff
@@ -110,10 +110,10 @@ fi
 /etc/xapi.d/plugins/ipxe
 /etc/xapi.d/plugins/kernel
 /etc/xapi.d/plugins/migration
-/etc/xapi.d/plugins/partition_utils.py
-/etc/xapi.d/plugins/pluginlib_nova.py
+/etc/xapi.d/plugins/partition_utils.py*
+/etc/xapi.d/plugins/pluginlib_nova.py*
```

Comment:
Actually, could you add .py, .pyo and .pyc so it doesn't include things like .py~?